tech/languages/csharp/csharp-ide.md

```diff
@@ -22,29 +22,30 @@ JetBrains Rider is full featured IDE for all things C#, and even F# on .NET Core
 
 [jetbrains.com/rider](http://jetbrains.com/rider)
 
-## MonoDevelop
-
-_Mono only_
+## Eclipse IDE (with plugin)
 
-MonoDevelop provides the best C# experience on Mono, the open-source multiplatform implementation of the .NET Framework.
+_.NET Core and Mono_
 
-[monodevelop.com](http://www.monodevelop.com)
+You can use the [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) on top of the established and popular [Eclipse IDE](/tools/eclipse/about.html). This C# extension for Eclipse IDE relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). It comes with complete edition features for C# in the Eclipse IDE. Relying on the Eclipse IDE provides all typical features of a good IDE (Polyglot development, SCM, deployment...) and give access to a rich ecosystem of extensions for about all existing technologies.
 
 ```
-$ sudo dnf install monodevelop
+$ sudo dnf install eclipse eclipse-mpc
 ```
+then install the aCute extension from [Eclipse Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
 
-## Eclipse IDE (with plugin)
+## MonoDevelop
 
-_Mono only, with upcoming support for .NET Core as well_
+_Mono only_
+
+MonoDevelop provides the best C# experience on Mono, the open-source multiplatform implementation of the .NET Framework.
 
-You can get [Eclipse IDE](/tools/eclipse/about.html) with [aCute plugin](https://github.com/mickaelistria/aCute), which relies on [OmniSharp](http://www.omnisharp.net) and [Language Server Protocol](https://github.com/Microsoft/language-server-protocol). Although it's still experimental, this already provides nice edition features for C# in the Eclipse IDE.
+[monodevelop.com](http://www.monodevelop.com)
 
 ```
-$ sudo dnf install eclipse eclipse-mpc
+$ sudo dnf install monodevelop
 ```
-then install the aCute extension from [Eclipse Marketplace](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental)
 
 ## Summary
 
-What would we recommend? JetBrains Rider is the best C# IDE, however it is not open source. VS Code is great for .NET Core and MonoDevelop for Mono. C# in Eclipse IDE is work in progress, and supports only Mono for now; it can already be useful if you're already an user of Eclipse IDE with any other programming language.
+What would we recommend? JetBrains Rider is the best C# IDE, however it is not open source. VS Code is great for .NET Core and MonoDevelop for Mono. C# in Eclipse IDE is still young and not very friendly to former Windows developer; but it can be useful if you're already an Eclipse IDE user.
```

Comment:
Please link to those `JetBrains` and `VS Code` (and I'd rather not use abbrev., unless they're official/marketed as such).

`'Windows developer; but ` - I'd prefer `,` instead of `;` here.

Reply:
`What would we recommend?`
I do not think you're answering the question in this block. Could it be formulated differently?
Or possibly do, instead of `summary`, something like:

## What IDE to use?
 - **JetBrains Rider** - blabla...
 - **VS Code** - blabla...
 . . .

Reply:
It's supposed to be a summary, the list of IDEs with description is above and it would seem weird to have two lists of IDE's..

Reply:
Well, my point is that it's just formated in a way that summary should be, but the content is just serialized list of IDEs.

Reply:
I don't think that we should really say "use this," so I'm trying to make it at least somewhat objective comparison with some pro's and con's. I don't know how else to achieve that...

Reply:
It's fine :) it was just an idea.
If you think of some change, please open new PR. 

I've got one additional question, though: is `VS Code` official name/short? Because otherwise it should be fixed.

Reply:
Well it's Visual Studio Code but yes, VS Code is kinda official name of the project, maybe not marketing name though. They're not very consistent with the different names... Their official repo is called `/etc/yum.repos.d/vscode.repo` then you have `dnf install code` and the official name is Visual Studio Code. It's kinda messy, but everyone is using VS Code so I think that it's fine.

https://code.visualstudio.com
https://github.com/Microsoft/vscode

Reply:
`kinda`
I'll fix it at least for title....